src/android/java/io/jxcore/node/BtConnectorHelper.java

```diff
@@ -68,30 +76,18 @@ public boolean isRunning(){
         }
     }
     public void Stop(){
```

Comment:
@DrJukka Why isn't this synchronized? You can still have a race condition where the same mBTConnector has its stop method called twice. Does that not matter?